Dockerfile

```diff
@@ -26,7 +26,7 @@ ADD https://github.com/WebAssembly/binaryen/archive/refs/tags/$BINARYEN_VERSION.
 # Adapted from https://github.com/WebAssembly/binaryen/blob/main/.github/workflows/build_release.yml
 RUN apk update && apk add build-base cmake git python3 clang ninja
 RUN tar -xf /tmp/binaryen.tar.gz
-RUN cd binaryen-version_*/ && cmake . -G Ninja -DCMAKE_CXX_FLAGS="-static" -DCMAKE_C_FLAGS="-static" -DCMAKE_BUILD_TYPE=Release -DBUILD_STATIC_LIB=ON && ninja wasm-opt
+RUN cd binaryen-version_*/ && git clone https://github.com/google/googletest.git ./third_party/googletest && cmake . -G Ninja -DCMAKE_CXX_FLAGS="-static" -DCMAKE_C_FLAGS="-static" -DCMAKE_BUILD_TYPE=Release -DBUILD_STATIC_LIB=ON && ninja wasm-opt
```

Comment:
Can we clone that with `--depth 1`?

It's not that much difference but still nice to have.